features/list_search.feature

```diff
@@ -0,0 +1,95 @@
+Feature: List Search
+    
+@Basic
+Scenario: 
+    Then I install the ccz app at "test_list_search.ccz"
+    Then I login with username "test_list_search" and password "123"
+    Then I scroll until I see the "Start" text
+    Then I touch the "Start" text    
```

Comment:
We have a custom command for this: `Then I press start` https://github.com/dimagi/commcare-ui-tests/blob/master/features/step_definitions/navigation.rb#L6-L8
which we can adjust over time to be more robust